lib/Elevate/YUM.pm

```diff
@@ -0,0 +1,66 @@
+package Elevate::YUM;
```

Comment:
The long term goal is to use `Cpanel::Pkgr` for all package manager related queries.  Right now, I'm just trying to get all our queries in 3 files:  `Elevate::RPM`, `Elevate::YUM`, and `Elevate::DNF` so we can verify that `Cpanel::Pkgr` meets all of our needs and so that we can convert everything to use `Cpanel::Pkgr` as systematically as possible.

FWIW: I'm not really adding any new code here.  Just factoring out the queries that we already make to their own files to make the queries easier to find and to make the existing code more dry.

I'm going to consider switching to use `Cpanel::Pkgr` out of scope for this change.